.gitignore

```diff
@@ -5,22 +5,20 @@
 
 # cargo
 target
-*/target
-*.rs.bk
 Cargo.lock
 
 # fuzz testing
-exonum/fuzz/target
 exonum/fuzz/corpus
 exonum/fuzz/artifacts
 
+# node
+node_modules
+
 # node (for cspell)
-/node_modules
 /npm-debug.log
 /package-lock.json
```

Comment:
Off-topic: for reproducible builds, it may make sense to include `package-lock.json` (and `package.json`) into the git tree, like [in the documentation repo](https://github.com/exonum/exonum-doc/blob/master/package.json).